lib/init.coffee

```diff
@@ -10,6 +10,8 @@ module.exports =
       title: 'gjslint executable path'
       default: 'gjslint'
     gjslintIgnoreList:
+      title: 'Errors to ignore'
```

Comment:
This needs much better wording, you shouldn't rely on users reading the README as most will not.